source/aks/upgrading-aks-clusters.html.md.erb

```diff
@@ -108,9 +110,13 @@ Some environments do have some additional requirements / checks that need to be
 We upgrade PTL/PTLSbox and Preview/Dev in-place, meaning we aren't building another cluster or switching in the appgw unless necessary. If one of these environments must instead be rebuilt and swapped over, please refer to the Rebuilding AKS Clusters guide.
 
 ### Ptlsbox, Ptl
-Make an announcement that Jenkins may be briefly unavailable in the cloud-native-announce slack channel. 
-This environment is best to be done early in the morning.
-See [Jenkins instances](../../jenkins/index.html) to check they are running after upgrades.
+
+This evironment is best to be done early in the morning. that Jenkins may be briefly unavailable on the #cloud-native-announce slack channel.
+Subsequently, please verify the status of [Jenkins instances] for CFT (https://build.hmcts.net/) and SDS (https://sds-build.hmcts.net/) to ensure they are operational following the upgrades. 
```

Comment:
```suggestion
Make an announcement that Jenkins may be briefly unavailable in the relevant cloud-native-announce slack channel. 
This environment is best to be done early in the morning.
Subsequently, please verify the status of [Jenkins instances] for CFT (https://build.hmcts.net/) and SDS (https://sds-build.hmcts.net/) to ensure they are operational following the upgrades. 
```

Reply:
This was suggested and advised so i made that change about this "being done early in". So i have moved to see this as the start.

Reply:
Fair, fixed a few typos:
```suggestion
This environment is best to be done early in the morning, make an announcement that Jenkins may be briefly unavailable on the relevant #cloud-native-announce slack channel.
Subsequently, please verify the status of [Jenkins instances] for CFT (https://build.hmcts.net/) and SDS (https://sds-build.hmcts.net/) to ensure they are operational following the upgrades. 
```

Reply:
This has been done, thanks